src/server/server.babel.js

```diff
@@ -1,12 +1,14 @@
-if (process.env.NODE_ENV !== 'production') {
-  if (!require('piping')({
-    hook: false,
-    ignore: /(\/\.|~$|\.json$)/i
-  })) {
-    return;
+(() => {
```

Comment:
I don't think this is quite Kosher. You're using an arrow function (ES6) in a file that needs to be straight up ecmascript 2015. I'd suggest you revert your changes and just inform the linter to ignore this file.